docs/tutorial/nfs-volume-resize.md

```diff
@@ -0,0 +1,166 @@
+# How to Expand/Resize NFS Volume
+
+OpenEBS dynamic-nfs-provisioner **doesn't support** natively expanding NFS volume but we can accomplish the same operation by following a workaround.
+
+
+For expanding NFS volume, you must ensure the following items(prerequisites) are taken care of:
+
+- The BackendStorageClass must support volume expansion, which can configure by editing the StorageClass definition to have `allowVolumeExpansion: true`
+
+  ```
+  kubectl patch sc openebs-lvmpv -p '{"allowVolumeExpansion": true}'
+  ```
+
+  ```sh
+  storageclass.storage.k8s.io/openebs-lvmpv patched
+  ```
+- NFS StorageClass must support volume expansion, which can configure by editing the NFS StorageClass defination to have `allowVolumeExpansion: true`
+
+  ```
```

Comment:
Will this work without requiring a Storageclass delete and re-add?

Reply:
Yes, We can update the field of allowVolumeExpansion without deletion of StorageClass